### ISSUE TITLE
Add SEs descriptions in the Statistical Methdology Update 

### ISSUE THREAD

**Issue by tzhangumd** [American-Institutes-for-Research/EdSurvey_A_Users_Guide#9]
@Sinan-Yavuz Can you add a description about how the SEs of the regression coefficients are calculated when PVs are predictors?

It can be in the same section, and be short if "11.2.7 Estimation of Standard Errors of Weighted Means When Plausible Values Are Present, Using the Jackknife Method" covers it.

Please also ask Paul to review the description before commit it to the branch.

**Comment by Sinan-Yavuz** [comment]
Yes, we are using the same methodology as 11.2.7. I will mention that in the text.

Is it OK, if I update the "statisticalMethodologyUpdate" branch?

**Comment by tzhangumd** [comment]
How about 11.2.8 Taylor series? Please also indicate if Taylor series works or not for this type of SEs. 

And could you share the texts with Paul and me here first?

**Comment by pdbailey0** [comment]
I do not plan to recode Taylor series for this, nor verify it for this. Emmanuel indicated we can de-emphasize Taylor series in EdSurvey in general since it is slower and more complicated.

**Comment by Sinan-Yavuz** [comment]
Sounds good! I will make a small adjustment for the JK method.

**Comment by Sinan-Yavuz** [comment]
I made two small adjustments and haven't pushed the changes yet. The changes are highlighted below:

![Screenshot 2023-11-10 at 11 46 24 AM](https://github.com/American-Institutes-for-Research/EdSurvey_A_Users_Guide/assets/47365408/dc949039-57fc-47e4-98c3-5110cfd6b694)

![Screenshot 2023-11-10 at 11 47 18 AM](https://github.com/American-Institutes-for-Research/EdSurvey_A_Users_Guide/assets/47365408/3dc64a44-c581-4319-9cf8-33cb2219a584)

**Comment by tzhangumd** [comment]
@Sinan-Yavuz Suggest editing the first one as "When the predictor and/or the outcome variable has plausible". The 2nd one looks good to me.
